x-pack/plugins/apm/public/components/shared/links/apm/transaction_detail_link/transaction_detail_max_groups_message.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiCode, EuiLink } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React from 'react';
+
+export const txGroupsDroppedBucketName = '_other';
+
+export function TransactionDetailMaxGroupsMessage() {
+  return (
+    <FormattedMessage
+      defaultMessage="The transaction group limit in APM Server has been reached. Excess groups have been captured in _other. Please see the {apmServerDocs} for {codeBlock} to increase this limit."
+      id="xpack.apm.transactionDetail.maxGroups.message"
+      values={{
+        apmServerDocs: (
+          <EuiLink
+            href={
+              'https://www.elastic.co/guide/en/apm/guide/current/transaction-metrics.html#transactions-max_groups'
```

Comment:
I'm not sure how I feel about linking to these docs. This config is only valid for APM Server standalone users, not Elastic agent users. That page also has no information for how to make that setting work in ESS (does it even?)

I think instead we should link to the Kibana docs, where we we can explain how to change the max number of transaction groups in all three APM Server/integration use cases.

Reply:
It looks like it is available in ESS (see above comment) but I can't find anything about the APM integration

Reply:
What if we link this to the troubleshooting section instead?

Reply:
but then in the troubleshooting we link to the APM documentation?

Reply:
Here's what I'm thinking:

UI links to the troubleshooting page. The troubleshooting page explains how to fix this problem if you're using...
* **APM Server standalone**: `aggregation.transaction.max_groups` and links to https://www.elastic.co/guide/en/apm/guide/current/transaction-metrics.html#transactions-max_groups'
* **ESS**: `apm-server.aggregation.transactions.*` and links to  https://www.elastic.co/guide/en/cloud/current/ec-manage-apm-settings.html
* **APM integration**: ? unkown setting name and links to APM integration docs if they exist

Reply:
We decided to move away from these settings and point now to memory limit.